remix.init/index.js

```diff
@@ -66,14 +66,13 @@ async function updatePackageJsonForEdge(directory) {
     // dev script is the same as the start script for Netlify Edge, "cross-env NODE_ENV=production netlify dev"
     scripts: {
       ...scripts,
-      dev: scripts["start"],
+      predev: "rimraf ./.netlify/edge-functions/",
```

Comment:
If someone runs `netlify serve` or `npm run start`, `./.netlify/edge-functions` remains. This causes issues if running `ntl dev` which now uses the Remix dev server which is why we delete it if it exists before running `netlify dev` which runs `npm run dev`.

Reply:
We already do this for the Netlify Functions template, but it deletes something different which is why we ovveride the `predev` script when using the Netlify Edge Functions template.